src/midgard/tiles.cc

```diff
@@ -299,6 +299,10 @@ template <class container_t>
 std::unordered_map<int32_t, std::unordered_set<unsigned short>>
 Tiles<coord_t>::Intersect(const container_t& linestring) const {
   std::unordered_map<int32_t, std::unordered_set<unsigned short>> intersection;
+  // protect against empty linestring
+  if (linestring.empty()) {
```

Comment:
a unit test was hitting this and ASAN caught it. if you look a few lines below there is a call to `linestring.front()` which of course is not valid if its empty